sdk/monitor/Azure.Monitor.OpenTelemetry.Exporter/src/Internals/Diagnostics/AzureMonitorExporterEventSource.New.cs

```diff
@@ -47,7 +47,7 @@ public void TransmissionFailed(bool fromStorage, int statusCode, ConnectionVars
             }
         }
 
-        [Event(8, Message = "{0} {1} {2}")]
+        [Event(8, Message = "{0} {1} {2}", Level = EventLevel.Warning)]
```

Comment:
If telemetry is dropped, is that an error or warning?

Reply:
this should be an error